pkg/azurediskplugin/main.go

```diff
@@ -53,6 +53,9 @@ var (
 	useCSIProxyGAInterface     = flag.Bool("use-csiproxy-ga-interface", true, "boolean flag to enable csi-proxy GA interface on Windows")
 	enableDiskOnlineResize     = flag.Bool("enable-disk-online-resize", true, "boolean flag to enable disk online resize")
 	allowEmptyCloudConfig      = flag.Bool("allow-empty-cloud-config", true, "Whether allow running driver without cloud config")
+	enableListVolumes          = flag.Bool("enable-list-volumes", false, "boolean flag to enable ListVolumes on controller")
+	enableListSnapshots        = flag.Bool("enable-list-snapshots", false, "boolean flag to enable ListSnapshots on controller")
+	restClientQPS              = flag.Int("client-qps", 15, "QPS for the rest client. Defaults to 15.")
```

Comment:
Let's use `kubeClientQPS` and `kube-client-qps` to more clearly indicate this is for K8s API access.